frontend/src/helpers/EnvoyHelpers.tsx

```diff
@@ -15,7 +16,7 @@ export const istioConfigLink = (halfQDN: string, objectType: string): JSX.Elemen
   if (nameParts.length === 2) {
     return (
       <React.Fragment>
-        <IstioObjectLink name={nameParts[0]} namespace={nameParts[1]} type={objectType}>
+        <IstioObjectLink name={nameParts[0]} namespace={nameParts[1]} cluster={HomeClusterName} type={objectType}>
```

Comment:
Should this be `HomeClusterName`?

Reply:
having this default value as a TODO to resolve later